website/content/en/docs/reference/metrics.md

```diff
@@ -6,197 +6,257 @@ weight: 7
 
 ### `karpenter_nodes_leases_deleted`
+
 Number of deleted leaked leases.
 
 ### `karpenter_nodes_eviction_queue_depth`
+
 The number of pods currently waiting for a successful eviction in the eviction queue.
 
 ### `karpenter_nodes_created`
+
 Number of nodes created in total by Karpenter. Labeled by owning nodepool.
 
 ### `karpenter_nodes_allocatable`
+
 Node allocatable are the resources allocatable by nodes.
 
 ## Pods Metrics
 
 ### `karpenter_pods_state`
+
 Pod state is the current state of pods. This metric can be used several ways as it is labeled by the pod name, namespace, owner, node, nodepool name, zone, architecture, capacity type, instance type and pod phase.
 
 ### `karpenter_pods_startup_time_seconds`
+
 The time from pod creation until the pod is running.
 
 ## Provisioner Metrics
 
 ### `karpenter_provisioner_scheduling_simulation_duration_seconds`
+
 Duration of scheduling simulations used for deprovisioning and provisioning in seconds.
 
 ### `karpenter_provisioner_scheduling_queue_depth`
+
 The number of pods currently waiting to be scheduled.
 
 ### `karpenter_provisioner_scheduling_duration_seconds`
+
 Duration of scheduling process in seconds.
 
 ## Nodeclaims Metrics
 
 ### `karpenter_nodeclaims_terminated`
+
 Number of nodeclaims terminated in total by Karpenter. Labeled by reason the nodeclaim was terminated and the owning nodepool.
 
 ### `karpenter_nodeclaims_registered`
+
 Number of nodeclaims registered in total by Karpenter. Labeled by the owning nodepool.
 
 ### `karpenter_nodeclaims_launched`
+
 Number of nodeclaims launched in total by Karpenter. Labeled by the owning nodepool.
 
 ### `karpenter_nodeclaims_initialized`
+
 Number of nodeclaims initialized in total by Karpenter. Labeled by the owning nodepool.
 
 ### `karpenter_nodeclaims_drifted`
+
 Number of nodeclaims drifted reasons in total by Karpenter. Labeled by drift type of the nodeclaim and the owning nodepool.
 
 ### `karpenter_nodeclaims_disrupted`
+
 Number of nodeclaims disrupted in total by Karpenter. Labeled by disruption type of the nodeclaim and the owning nodepool.
 
 ### `karpenter_nodeclaims_created`
+
 Number of nodeclaims created in total by Karpenter. Labeled by reason the nodeclaim was created and the owning nodepool.
 
 ## Interruption Metrics
 
 ### `karpenter_interruption_received_messages`
+
 Count of messages received from the SQS queue. Broken down by message type and whether the message was actionable.
 
+**Note**: The `karpenter_interruption_received_messages` metric is aggregated at the account level. Therefore, it may include messages that are not directly related to Karpenter. If you notice a discrepancy between this metric and the `karpenter_nodeclaims_terminated{reason="interruption"}` metric, it could be due to the inclusion of non-Karpenter-related events in the `karpenter_interruption_received_messages` metric. To ensure accurate tracking, consider filtering out non-Karpenter-related events.
```

Comment:
This file is auto-generated, so if you want to add custom text to this file, we are going to have to make changes to the `metrics_gen_docs.go` script that is responsible for generating all of this info.

Reply:
On a separate note (since I'm assuming you are making this note because you were trying to make an alert based off of this metric), is there a way that you think that we could make this metric more helpful for monitoring?